doc/source/whatsnew/v1.1.0.rst

```diff
@@ -36,6 +36,33 @@ For example:
    ser["2014"]
    ser.loc["May 2015"]
 
+.. _whatsnew_110.key_sorting:
+
+Sorting with keys
+^^^^^^^^^^^^^^^^^
+
+We've added a ``key`` argument to the DataFrame and Series sorting methods, including
+:meth:`DataFrame.sort_values`, :meth:`DataFrame.sort_index`, :meth:`Series.sort_values`,
+and :meth:`Series.sort_index`. The ``key`` can be any callable function which is applied
+to the each column of a DataFrame before sorting is performed (:issue:`27237`).
```

Comment:
Just for the sake of explicitness, can you add an example here for DataFrame along with Series? I think helpful to highlight the direction in which the sort is applied

Reply:
Added an example for a DataFrame and updated the What's New.